packages/ui/src/components/Logo/Logo.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import blackHorizontalLogo from '../../../images/LogoBlackHorizontal.svg';
+import blackSquareLogo from '../../../images/LogoBlackSquare.svg';
+import whiteHorizontalLogo from '../../../images/LogoWhiteHorizontal.svg';
+import whiteSquareLogo from '../../../images/LogoWhiteSquare.svg';
+import { Logo } from './Logo';
+
+describe('Logo', () => {
+  it('renders default', () => {
+    render(<Logo />);
+
+    const logo = screen.getByAltText('Logo CodersCamp') as unknown as HTMLImageElement;
```

Comment:
This double casting doesn't seem right. What is the reasoning behind it?

Reply:
fount a workaround for this file - casting removed

Reply:
fount a workaround for this file - casting removed